src/main/java/org/icgc/argo/workflow_management/service/NextflowService.java

```diff
@@ -62,19 +62,19 @@
 @Service(value = "nextflow")
 public class NextflowService implements WorkflowExecutionService {
 
-  /** Constants */
+  // Constants
```

Comment:
disturbing :stuck_out_tongue:

Reply:
the /* triggers a format rule that says it must apply to only the first thing under, but we want that to apply to the group sooo yeah